src/project/project.cpp

```diff
@@ -397,25 +400,61 @@ namespace gs::management {
         auto project = std::make_shared<gs::management::Project>(true);
 
         project->setProjectName(project_name);
-        if (data.project_path.extension() != Project::EXTENSION) {
-            std::cerr << std::format("project_path must be {} file: {}", Project::EXTENSION, data.project_path.string()) << std::endl;
+        if (data.output_path.empty()) {
+            LOG_ERROR("output_path is empty");
+            return nullptr;
+        }
+        std::filesystem::path project_path = data.project_path;
+        if (project_path.empty()) {
+            project_path = data.output_path / "project.ls";
+            LOG_ERROR("project_path is empty - creating new project.ls file");
```

Comment:
- stop_refine_scale2d
[20:15:09.083] [info] main.cpp:34 ========================================
[20:15:09.083] [info] main.cpp:35 3D Gaussian Splatting CUDA
[20:15:09.083] [info] main.cpp:36 ========================================
Starting headless training...
[20:15:09.083] [error] project.cpp:410 project_path is empty - creating new project.ls file
[20:15:09.084] [info] loader_service.cpp:60 Using COLMAP loader for: data/bicycle
[  0.0%] Loading COLMAP dataset...

That's not an error. Just info, right?